core/src/main/java/apoc/schema/Schemas.java

```diff
@@ -110,6 +110,21 @@ public Boolean constraintExistsOnRelationship(@Name("type") String type, @Name("
         return result;
     }
 
+    private boolean checkIfConstraintNotExists(String label, Map<String, List<Object>> constraints, AssertSchemaResult info) {
+        if (constraints.containsKey(label)) {
+            return !constraints.get(label).removeIf(item -> {
+                if (item instanceof String) {
+                    return item.equals(info.key);
+                } else {
+                    Collections.sort(info.keys);
+                    Collections.sort((List) item);
+                    return info.keys.equals(item);
```

Comment:
Se `info.keys` e `item` non contengono doppioni (credo di no) fai cosi:
```suggestion
                    Set<Object> keys = new HashSet<>(info.keys);
                    Set<Object> items = new HashSet<>(items);
                    return keys.equals(item);
```